source/extensions/clusters/redis/redis_cluster_lb.h

```diff
@@ -67,6 +67,16 @@ class RedisLoadBalancerContext {
 class RedisLoadBalancerContextImpl : public RedisLoadBalancerContext,
                                      public Upstream::LoadBalancerContextBase {
 public:
+  /**
+   * The load balancer context for Redis requests. Note that use_crc16 implies using Redis cluster
```

Comment:
The fact that use_crc16 implied redis cluster is kind of confusing. Can you potentially rename this variable in some way to make this more clear? It seems like this type of logic is better handled at the caller?

Reply:
yes, the naming is a bit confusing.  Let me change that and update the comments.  Having this logic here is easier to test than in connection pool.